pkg/image/api/helper.go

```diff
@@ -238,43 +281,81 @@ func NormalizeImageStreamTag(name string) string {
 
 // ImageWithMetadata returns a copy of image with the DockerImageMetadata filled in
 // from the raw DockerImageManifest data stored in the image.
-func ImageWithMetadata(image Image) (*Image, error) {
+func ImageWithMetadata(image *Image) (*Image, error) {
```

Comment:
This function now mutates the original value that as passed in.  It fans out more than one function call.  Can you confirm you've completed the search?